### PR TITLE
disable Next Level button until sandbox is ready

### PR DESCRIPTION
**Pull request by lesnitsky**

### CODE REVIEW

File: src/game-master/middlewares/phoenix-gm-middleware.js [hunk 1]
```diff
@@ -60,6 +63,8 @@ function handleServerMessage(store, message) {
 
     switch (message.name) {
         case MESSAGE_NAME.gameMasterSessionState:
+            store.dispatch(setReadyState(message.sandboxStatus === ReadyState.READY));
```
Comment (kirilknysh) [inline src/game-master/middlewares/phoenix-gm-middleware.js]: GM session state has `sandboxStatus` property

Comment (lesnitsky) [reply src/game-master/middlewares/phoenix-gm-middleware.js]: it is handle below

Comment (lesnitsky) [reply src/game-master/middlewares/phoenix-gm-middleware.js]: or am I missing smth

Comment (kirilknysh) [reply src/game-master/middlewares/phoenix-gm-middleware.js]: My fault, stupid I am

**Pull request merged**